sitecore-xp0-sxa-horizon/docker/docker-compose.override.yml

```diff
@@ -0,0 +1,148 @@
+  solr-init:
+    image: ${REGISTRY}${COMPOSE_PROJECT_NAME}-xp0-solr-init:${VERSION:-latest}
+    build:
+      context: ./build/solr-init
+      args:
+        BASE_IMAGE: ${SITECORE_DOCKER_REGISTRY}sitecore-xp0-solr-init:${SITECORE_VERSION}
+        SXA_IMAGE: ${SITECORE_MODULE_REGISTRY}sxa-xp1-assets:${SXA_VERSION}
+
+  cm:
+    image: ${REGISTRY}${COMPOSE_PROJECT_NAME}-xp0-cm:${VERSION:-latest}
+    build:
+      context: ./build/cm
+      args:
+        BASE_IMAGE: ${SITECORE_DOCKER_REGISTRY}sitecore-xp0-cm:${SITECORE_VERSION}
+        SPE_IMAGE: ${SITECORE_MODULE_REGISTRY}spe-assets:${SPE_VERSION}
+        SXA_IMAGE: ${SITECORE_MODULE_REGISTRY}sxa-xp1-assets:${SXA_VERSION}
+        HRZ_IMAGE: ${SITECORE_MODULE_REGISTRY}horizon-integration-xp1-assets:${HRZ_VERSION}
+        TOOLING_IMAGE: ${SITECORE_TOOLS_REGISTRY}sitecore-docker-tools-assets:${TOOLS_VERSION}
+    environment:
+      SITECORE_LICENSE_LOCATION: C:\license\license.xml
+      SITECORE_DEVELOPMENT_PATCHES: DevEnvOn,CustomErrorsOff,DebugOn,DiagnosticsOff,InitMessagesOff,RobotDetectionOff
+      Sitecore_Horizon_ClientHost: https://${HRZ_HOST}
+    volumes:
+      - .\data\cm\website:C:\deploy
+      - .\data\cm\logs:C:\inetpub\wwwroot\App_Data\logs
+      - ${LICENSE_PATH}:C:\license
+    entrypoint: powershell -Command "& C:\tools\entrypoints\iis\Development.ps1"
+    mem_limit: ${MEM_LIMIT_CM:-4GB}
+    restart: always
+    labels:
+      - "traefik.http.middlewares.redirect-to-https.redirectScheme.scheme=https"
+      - "traefik.http.routers.cm.entrypoints=web"
+      - "traefik.http.routers.cm.rule=Host(`${CM_HOST}`)"
+      - "traefik.http.routers.cm.middlewares=redirect-to-https"
+
+  id:
+    environment:
+      Sitecore_Sitecore__IdentityServer__Clients__DefaultClient__AllowedCorsOrigins__AllowedCorsOriginsGroup2: https://${HRZ_HOST}
+      SITECORE_LICENSE_LOCATION: C:\license\license.xml
+    volumes:
+      - ${LICENSE_PATH}:C:\license
+
+  xconnect:
+    image: ${REGISTRY}${COMPOSE_PROJECT_NAME}-xp0-xconnect:${VERSION:-latest}
+    build:
+      context: ./build/xconnect
+      args:
+        BASE_IMAGE: ${SITECORE_DOCKER_REGISTRY}sitecore-xp0-xconnect:${SITECORE_VERSION}
+        TOOLING_IMAGE: ${SITECORE_TOOLS_REGISTRY}sitecore-docker-tools-assets:${TOOLS_VERSION}
+    environment:
+      SITECORE_LICENSE_LOCATION: C:\license
+    volumes:
+      - .\data\xconnect:C:\deploy
+      - ${LICENSE_PATH}:C:\license
+    entrypoint: powershell -Command "& C:\tools\entrypoints\iis\Development.ps1"
+
+  xdbsearchworker:
+    environment:
+      SITECORE_LICENSE_LOCATION: C:\license
+    volumes:
+      - ${LICENSE_PATH}:C:\license
+
+  xdbautomationworker:
+    environment:
+      SITECORE_LICENSE_LOCATION: C:\license
+    volumes:
+      - ${LICENSE_PATH}:C:\license
+
+  cortexprocessingworker:
+    environment:
+      SITECORE_LICENSE_LOCATION: C:\license
+    volumes:
+      - ${LICENSE_PATH}:C:\license
+
+  horizon:
```

Comment:
Can we rename this back to `hrz` so it matches the container example from Sitecore (even thought I prefer calling it horizon 😛)